src/Nordea/Components/Chat.elm

```diff
@@ -194,49 +211,52 @@ chatHistoryView attrs { sentFrom, sentAt, sender, message, isIncomingMessage, re
             Text.textTinyLight
                 |> Text.view (css [ color Colors.darkGray, whiteSpace noWrap ] :: attr) [ Html.text text ]
     in
-    Html.column (css [ gap (rem 0.25) ] :: attrs)
-        [ Html.row [ css [ justifyContent spaceBetween ] ]
-            [ messageLabel [ css [ marginRight (rem 0.25), textOverflow ellipsis, overflow hidden ] ] sentFrom
-            , messageLabel [] sentAt
-            ]
-        , Html.row [ css [ alignItems center, justifyContent spaceBetween ] ]
-            [ sender
-                |> Html.viewMaybe
-                    (\sender_ ->
-                        Text.textTinyHeavy |> Text.view [] [ Html.text sender_ ]
-                    )
-            , messageChipText
-                |> Html.viewMaybe
-                    (\messageChipText_ ->
-                        Text.textTinyLight
-                            |> Text.view
-                                [ css
-                                    [ display inlineBlock
-                                    , borderRadius (rem 1.25)
-                                    , padding2 (rem 0.125) (rem 0.5)
-                                    , backgroundColor Colors.lightBlue
-                                    , textOverflow ellipsis
-                                    , overflow hidden
-                                    , whiteSpace noWrap
-                                    ]
-                                ]
-                                [ Html.text messageChipText_ ]
-                    )
-            ]
-        , Html.column [ class "chat-message", css [ padding (rem 0.625), gap (rem 0.625), messageStyles ] ]
```

Comment:
@anisha-deverashetty , do you know what this class is used for, and if it is ok to remove?

Reply:
I see, used by FF